homeassistant/helpers/config_validation.py

```diff
@@ -291,7 +292,7 @@ def url(value: Any) -> str:
     raise vol.Invalid('invalid url')
 
 
-def ordered_dict(value_validator, key_validator=match_all):
+def ordered_dict(value_validator, key_validator=match_all, domain=None):
```

Comment:
The validators itself should not be in charge of logging errors. It should just blow up and whoever was validating using this validator will have to handle the exception.